tests/manage/pv_services/test_pvc_delete_verify_size_is_returned_to_backendpool.py

```diff
@@ -30,18 +30,18 @@
 
 
 @retry(UnexpectedBehaviour, tries=20, delay=5, backoff=1)
-def check_ceph_used_space():
+def check_ceph_used_space(cbp_name):
```

Comment:
@parikshithb This function needs to be moved to cluster.py.
I've suggested this in https://github.com/red-hat-storage/ocs-ci/pull/728/files#diff-e2c2dcfab854db263b9a45a42d6f4b55R388

Thinking maybe to do that in a different PR

Reply:
moved